randomActivityGen.py

```diff
@@ -240,7 +240,9 @@ def main():
         setup_schools(net, stats, int(args["--schools.count"]))
 
     # Write statistics back
-    stats.write(args["--output-file"])
+    output_file = args["--output-file"]
+    os.mkdir(os.path.dirname(output_file))
```

Comment:
Will this not throw an error if the directory already exists?

Reply:
Right. This function is not smart .. Fixed in 937c99b

Reply:
Approved